### ISSUE TITLE
How to get SHREC16 dataset ?

### ISSUE THREAD

**Issue by cuge1995** [liruihui/PointAugment#12]
I've download `shapenetcore-v2.zip`, but how to separate it into train/test set ?

**Comment by liruihui** [comment]
FYI,  SHREC16 is from  http://shapenet.cs.stanford.edu/shrec16/

**Comment by cuge1995** [comment]
As described in [http://shapenet.cs.stanford.edu/shrec16/](http://shapenet.cs.stanford.edu/shrec16/), `We use the ShapeNetCore subset of ShapeNet which contains about 51,300 3D models over 55 common categories, each subdivided into several subcategories. We created a 70%/10%/20% training/validation/test split from this dataset.`

Therefore, it should be 35,708 (70%) for training, 5,158 (10%) shapes for validation and 10,261 (20%) shapes for testing. (This is suggested by [PointCloudDatasets](https://github.com/AnTao97/PointCloudDatasets), they claimed that in official document there should be 51,190 shapes in total, but 63 shapes are missing in original downloaded ShapeNetCore.v2 dataset)

I was wondering how to fair compare with your's result by using [PointCloudDatasets](https://github.com/AnTao97/PointCloudDatasets), should I use the training set (35,708) for training and the validation set(5158) for testing? Or use the testing set(10261) for testing?
